### ISSUE TITLE
Permission error on macOS

### ISSUE THREAD

**Issue by drallgood** [rancher-sandbox/rancher-desktop#1441]
### Rancher Desktop Version

1.0.0

### Rancher Desktop K8s Version

1.23.3

### What operating system are you using?

macOS

### Operating System / Build Version

macOS Monterey 12.3 Beta

### What CPU architecture are you using?

arm64 (Apple Silicon)

### Actual Behavior

After a clean install, rancher fails to start K8S because a directory it needs doesn't have write permissions:
/private/var/run/rancher-desktop-lima

### Steps to Reproduce

Install rancher Desktop on a clean macOS

### Result

```
2022-02-01T15:53:34.731Z: Running command limactl list --json...
2022-02-01T15:53:34.810Z: Running command limactl start --tty=false /Users/patriceb/Library/Application Support/rancher-desktop/lima/_config/0.yaml...
time="2022-02-01T16:53:34+01:00" level=info msg="Terminal is not available, proceeding without opening an editor"
time="2022-02-01T16:53:34+01:00" level=info msg="Starting switch daemon for \"bridged_en0\" network"
time="2022-02-01T16:53:34+01:00" level=fatal msg="\"/private/var/run/rancher-desktop-lima\" doesn't seem to be writable by the daemon (gid:1) group"
2022-02-01T15:53:34.926Z: + limactl start --tty=false /Users/patriceb/Library/Application Support/rancher-desktop/lima/_config/0.yaml
2022-02-01T15:53:34.927Z: Error: /Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl exited with code 1
2022-02-01T15:53:34.927Z: Error starting lima: Error: /Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl exited with code 1
    at ChildProcess.<anonymous> (/Applications/Rancher Desktop.app/Contents/Resources/app.asar/dist/app/background.js:1:8656)
    at ChildProcess.emit (node:events:394:28)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12)

```

### Expected Behavior

No errors

### Additional Information

Fix: `sudo chmod 775 /private/var/run/rancher-desktop-lima`

How it looked like before:
<img width="592" alt="Screen Shot 2022-02-01 at 5 16 11 PM" src="https://user-images.githubusercontent.com/127142/152006680-6df584ea-7835-4d2f-a9e7-26aca63a38b1.png">

**Comment by evertonlperes** [comment]
Thanks for filing the issue @drallgood .
We have a [Epic](https://github.com/rancher-sandbox/rancher-desktop/issues/1224) that we're tracking down issues related to sudo and permissions. I'll tag the epic here for your reference.

**Comment by dwaite** [comment]
The permission on this directory appears to be reset periodically, possibly on macOS upgrade. On my machine which runs beta macOS versions, the issue is encountered frequently.

**Comment by mmatczuk** [comment]
Thanks, I have hit the same after OS upgrade. Changing permissions helped.

**Comment by jandubois** [comment]
This has been fixed in the 1.2.0 release.
